tests/utils/testrunner.go

```diff
@@ -0,0 +1,73 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package utils
+
+import (
+	kube "github.com/dapr/dapr/tests/platforms/kubernetes"
+)
+
+// runnerFailExitCode is the exit code when test runner setup is failed
+const runnerFailExitCode = 1
+
+// testingMInterface interface is used for testing TestRunner
+type testingMInterface interface {
+	Run() int
```

Comment:
why not return as 'error' ?

Reply:
Please see this - https://golang.org/pkg/testing/#hdr-Main 

This is the interface for testing.M's Run method.

Reply:
Won't fix.

Reply:
this is by-design. won't fix.